src/plugins/ipmi_intf.c

```diff
@@ -399,6 +409,21 @@ ipmi_intf_socket_connect(struct ipmi_intf * intf)
 			continue;
 		}
 
+		if (params->bind_intf != NULL) {
+			#ifdef SO_BINDTODEVICE
+			int err = setsockopt(intf->fd, SOL_SOCKET, SO_BINDTODEVICE,
+					params->bind_intf, strlen(params->bind_intf) + 1);
+			if (err) {
+				lprintf(LOG_ERR, "Binding to network interface %s failed: %d",
+						params->bind_intf, err);
+				continue;
+			}
+			#else
+			lprintf(LOG_ERR, "Binding network interface is not supported on this platform.");
```

Comment:
I'd prefer the whole scope of changes related to this option to be left out if it is not supported, rather than listing the option and then printing that it is not supported.

Reply:
Done.